Cargo.toml

```
@@ -6,13 +6,26 @@ license = "MIT"
 authors = ["Gear Technologies"]
 
 [dependencies]
-gstd = { git = "https://github.com/gear-tech/gear.git" , branch = "stable" }
+gstd = { git = "https://github.com/gear-tech/gear.git" }
 ft-io = { path = "io" }
 hashbrown = "0.13.1"
+gmeta = { git = "https://github.com/gear-tech/gear", features = ["codegen"] }
```

```suggestion
gmeta = { git = "https://github.com/gear-tech/gear" }
```
See the previous comment.